test/compress/keep_names.js

```diff
@@ -88,6 +88,33 @@ keep_some_fnames: {
   }
 }
 
+keep_some_fnames_reduce: {
+  options = {
+    reduce_vars: true,
+    unused: true,
+    keep_fnames: /Element$/,
+  };
```

Comment:
Please remove this semicolon. Although technically ES5, we treat the test files as a DSL.